charms/kfp-persistence/src/charm.py

```diff
@@ -122,6 +122,8 @@ def _get_interfaces(self):
             raise CheckFailedError(str(err), WaitingStatus)
         except NoCompatibleVersions as err:
             raise CheckFailedError(str(err), BlockedStatus)
+        except RelationDataError as err:
+            raise CheckFailedError(str(err), BlockedStatus)
```

Comment:
are you sure it is Blocked?

what can user do to unblock it?

Reply:
I'm not, it's just what we do in the KFP API charm: https://github.com/canonical/kfp-operators/blob/b19cf459b6974e839280f52501b3268ad37647d7/charms/kfp-api/src/charm.py#L338-L339

Reply:
> related?

I don't know, there's nothing on that issue regarding what exactly happened and how you ended up there 😛

Reply:
What would you expect the status to be there?

Reply:
we cannot just set waiting without issue handling, since what will we wait :)

Reply:
Will add some context in the linked issue. Let's keep this as is for now, until we have a solid solution.